src/scripts/main.js

```diff
@@ -24,15 +24,15 @@ const firstPromise = new Promise((resolve, reject) => {
 });
 
 const secondPromise = new Promise((resolve, reject) => {
-  document.addEventListener('click', () => {
+  document.addEventListener('mouseup', () => {
```

Comment:
`mouseup` is faster than `click`, and if click is within `firstPromise`, there could be an issue where `secondPromise` completes before `firstPromise`. However, I cannot see the web page, I am not certain.